pom.xml

```diff
@@ -207,11 +287,39 @@
         <dependency>
             <groupId>com.azure</groupId>
```

Comment:
all com.azure dependencies shouldnt specify a version so the azure bom management versions take effect

Reply:
You're right , unfortunately when we use the BOM, there are vulnerabilities that come up in submodules :( . Confluent folks sent us emails about these vulnerabilities  hence had to exclude and include with later versions manually

Reply:
Did you try with latest bom?

Reply:
Hello Ohad, yes we did..we use 1.2.11 (latest version)

Reply:
1.2.12 is now released

Reply:
Updated the release for this BOM